### PR TITLE
Document dependency python-wxGTK for mayavi visualizer.

### LINKED ISSUE #4001: Mayavi visualizer depends on python3-wxgtk4
To get rid of
No pyface toolkit plugin for wx
upon 
`import espressomd.visualization_mayavi`

### PR DESCRIPTION
**Pull request by pkreissl**
Fixes #4001

### CODE REVIEW

**Comment by KaiSzuttor** [comment]
is it in ```requirements.txt```?

**Comment by pkreissl** [comment]
> is it in `requirements.txt`?

No, it is not. Neither are the vtk python bindings. If you want these to be added to `requirements.txt`, which version would you go for? The one currently available via the standard Ubuntu 20.04 repository? The Python 3 or Python 2 versions? In Fedora, e.g., the packages are called differently -- so pip instead?

Or do we keep it at this general level of: ok, you need `vtk` and `wxGTK4` pythonXY bindings.

I would be fine with all versions.

**Comment by KaiSzuttor** [comment]
just put it in the requirements file without any version, as long as we are not aware of any version that does not work, we can just omit the version.

File: requirements.txt [hunk 1]
```diff
@@ -9,7 +9,9 @@ pint>=0.9
 matplotlib>=2.1.1
 vtk>=7.1.1
 PyOpenGL>=3.1.0
-mayavi>=4.5.0
+mayavi>=4.5.0 # for mayavi-visualizer
+python-vtk # for mayavi-visualizer, e.g. python3-vtk7 (Ubuntu)
+python-wxgtk # for mayavi-visualizer, e.g. python3-wxgtk4.0 (Ubuntu)
```
Comment (jngrad) [inline requirements.txt]: there is no package `python-wxgtk` on pypi (https://pypi.org/project/python-wxgtk/), please remove it

`python-vtk` is a personal project with no documentation nor repository (https://pypi.org/project/python-vtk/), which is then used with `import python_vtk`; are you really sure this is what we should install?

**Review by jngrad — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (jngrad) [reply requirements.txt]: I removed that change to `requirements.txt`.

**Review by jngrad — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (pkreissl) [reply requirements.txt]: Fine with me, thanks!

**Pull request merged**